app/src/main/java/de/westnordost/streetcomplete/quests/fuel_service/AddFuelSelfService.kt

```diff
@@ -0,0 +1,41 @@
+package de.westnordost.streetcomplete.quests.fuel_service
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CAR
+import de.westnordost.streetcomplete.ktx.toYesNo
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddFuelSelfService : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          amenity = fuel
+          and !self_service
+          and !automated
+    """
+    override val commitMessage = "Add self service information to fuel station"
+    override val wikiLink = "Tag:amenity=fuel"
+    override val icon = R.drawable.ic_quest_fuel_self_service
+
+    override val questTypeAchievements = listOf(CAR)
+
+    override fun getTitle(tags: Map<String, String>) : Int {
+        val hasName = tags.containsKey("name")
+        val hasBrand = tags.containsKey("brand")
+        return when {
+            hasName || hasBrand ->  R.string.quest_fuelSelfService_name_title
+            else ->                 R.string.quest_fuelSelfService_title
+        }
+    }
+
+    override val enabledInCountries = NoCountriesExcept("IT")
```

Comment:
Because of https://github.com/streetcomplete/StreetComplete/pull/2822#issuecomment-977850275, maybe it should be enabled in the UK?

```suggestion
    override val enabledInCountries = NoCountriesExcept("IT", "UK")
```

Reply:
Mh, I did not understand if it's really useful or not as it is now, @timothywashere what do you think?

Reply:
> Mh, I did not understand if it's really useful or not as it is now, @timothywashere what do you think?

I think it's useful and would like to see it.

Despite the fact that I used the phrase earlier, "Pump your own gas" isn't really appropriate in British English, as an Americanism it would offend a lot of people. I'm not sure what is a succinct correct phrase, "Dispense your own fuel?" I suppose.

If anyone objects to this quest in the UK an option would be to only shown the quest for small or unknown brands. You wouldn't get self-dispense at BP/Shell/Esso/Tesco/Morrisons/Asda/Sainsburys. But I am happy to answer this quest for all petrol stations so don't add that functionality unless someone else complains.

Reply:
Also one pedantic query. Should it be "Can you.." or "Must you.." . Can implies a choice that isn't there (In the UK anyway).
For self service stations it's usually required that you operate the pump yourself, unless you are disabled and may be able to ask for help.
If it's not self service, then you are forbidden from operating the pump.

Reply:
@timothywashere Thank you for the answer, not being English I'm unaware of what's correct and what isn't and can't appreciate the nuances between two wordings, if you can suggest a question for the tag I'd be very glad.

I added UK by the way, thank you!

Reply:
> If anyone objects to this quest in the UK an option would be to only shown the quest for small or unknown brands. You wouldn't get self-dispense at BP/Shell/Esso/Tesco/Morrisons/Asda/Sainsburys.

I don't know how this could be handled, maybe just remove all fuels with brand tags altogether?